RecipesLyricsMashup/RecipesLyricsMashupV2/RecipeLyricsMashup/RecipeLyricsMashup/Pages/Index.cshtml

```diff
@@ -3,9 +3,12 @@
 @{
     ViewData["Title"] = "Home page";
     var recipePuppyJson2 = ViewData["recipePuppyJson2"];
-    QuickTypeRecipe.Result[] recipePuppyJson = (QuickTypeRecipe.Result[]) ViewData["recipePuppyJson"];
+    // QuickTypeRecipe.Result[] recipePuppyJson = (QuickTypeRecipe.Result[]) ViewData["recipePuppyJson"];

```

Comment:
Remove commented out code.  It's considered bad practice to commit commented-out code to version control, because you can always go back in history to see what used to be there.